src/corva/runners/task.py

```diff
@@ -1,12 +1,18 @@
-from typing import Callable
+import enum
+from typing import Any, Callable
 
 import requests
 
 from corva.api import Api
-from corva.models.task import TaskContext, TaskEvent, TaskStatus
+from corva.models.task import TaskContext, TaskEvent
 
 
-def get_task_data(api: Api, task_id: str) -> TaskEvent:
+class TaskStatus(enum.Enum):
+    fail = 'fail'
```

Comment:
I don't know if this matters, but the actual task status is `failed` or `succeeded`. The endpoints end with /fail or /success.

Reply:
Thanks, we don't parse actual task statuses, only set new ones using `v2/tasks/{task_id}/fail` and `v2/tasks/{task_id}/success`